### PR TITLE
various: deprecate

### PR DESCRIPTION
**Pull request by SMillerDev**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----
Based on the issues in: https://github.com/Homebrew/homebrew-core/issues/139929

### CODE REVIEW

File: Formula/l/lha.rb [hunk 1]
```diff
@@ -1,22 +1,11 @@
 class Lha < Formula
   desc "Utility for creating and opening lzh archives"
-  homepage "https://lha.osdn.jp/"
-  # Canonical: https://osdn.net/dl/lha/lha-1.14i-ac20050924p1.tar.gz
-  url "https://dotsrc.dl.osdn.net/osdn/lha/22231/lha-1.14i-ac20050924p1.tar.gz"
-  version "1.14i-ac20050924p1"
+  homepage "https://github.com/jca02266/lha/"
+  url "https://github.com/jca02266/lha/archive/refs/tags/release-20211125.tar.gz"
+  version "1.14i-ac20211125"
   sha256 "b5261e9f98538816aa9e64791f23cb83f1632ecda61f02e54b6749e9ca5e9ee4"
```
Comment (Bo98) [inline Formula/l/lha.rb:6]: This looks to be a version bump so would be worth doing this a separate PR so we don't publish bottles for the rest.

File: Formula/p/phpbrew.rb [hunk 2]
```diff
@@ -2,7 +2,7 @@ class Phpbrew < Formula
   desc "Brew & manage PHP versions in pure PHP at HOME"
   homepage "https://phpbrew.github.io/phpbrew"
   url "https://github.com/phpbrew/phpbrew/releases/download/2.2.0/phpbrew.phar"
-  sha256 "d1f248d486d6d2368d67cf0bb79371ee0abfc7e566bfc279e750372d1b0f96bd"
+  sha256 "3247b8438888827d068542b2891392e3beffebe122f4955251fa4f9efa0da03d"
```
Comment (Bo98) [inline Formula/p/phpbrew.rb:5]: Again probably best as a separate PR since this needs new bottles (and maybe a revision bump?).

**Review by iMichka — APPROVED** [review]
